src/qt/radiobut.cpp

```diff
@@ -59,3 +72,38 @@ QWidget *wxRadioButton::GetHandle() const
 {
     return m_qtRadioButton;
 }
+
+void wxRadioButton::CreateAndJoinNewGroup()
+{
+    // Note that the QButtonGroup created below will be deallocated by the QRadioButton owing
+    // to using that as its parent.
+    QButtonGroup* qtButtonGroup = new QButtonGroup( GetHandle() );
+    qtButtonGroup->addButton( m_qtRadioButton );
+}
+
+void wxRadioButton::SearchForPreviousGroupToJoin( wxWindow *parent )
+{
+    wxWindowList::compatibility_iterator nodeThis = parent->GetChildren().Find(this);
+
+    for ( wxWindowList::compatibility_iterator node = nodeThis->GetPrevious(); node; node = node->GetPrevious() )
```

Comment:
```suggestion
    for ( wxWindow* previous = GetPrevSibling(); previous; previous = GetPrevSibling() )
```
This way you won't need to pass `parent` as a parameter either.